Islandora/islandora_defaults
# Clean up islandora_defaults so it's "clean" after install

**GitHub Issue**: (link)

Creating pull requests to Islandora Defaults has been described as "trivial". However, the difference between the exported feature code after a full install, and the code in the repository, is significant.

If we should be able to create PRs by exporting the feature code in a running instance, then we need to reconcile these changes that are present, even when no changes have been made.

# What does this Pull Request do?

Reconciles islandora_defaults code with the results of exporting the islandora_defaults feature.

# What's new?

* file modes changed
* UUIDs removed
* context logic operators added
* new sections like 'status' for taxonomy terms
* submodules removed

# How should this be tested?

Sorry to be snarky, but I honestly wish I knew. Technically if you were able to create an islandora without Defaults, then install this version, it should... work? 

# Additional Notes:

If "export from the Features page" is a desired workflow, we're going to have to do something about the submodules.

# Interested parties
@dannylamb  @Islandora/8-x-committers
